### ISSUE TITLE
Make Linux requirements more explicit

### ISSUE THREAD

**Issue by lbogaardt** [cltl/vu-rm-pip3#12]
The '_svm_wsd/blob/master/install_naf.sh_' script requires '_unzip_'. This does not (always) come by default with Linux/Ubuntu. The '_install.sh_' script also does not provide a clear error message if the user does not have '_unzip_'. Therefore, it may be necessary for the user to run 'apt-get install unzip'. This should be made more explicit (in a list of requirements or by using a script to install all those requirements automatically).

Additional Linux requirements are:
- java (v8)
- maven (v3)
- python (v3)
- pip (v3)
- timbl
- libxss1
- libxft2
- libtk8.5
- unzip

**Comment by sarnoult** [comment]
Requirements are better documented now, but this is still subject to improvement.

**Comment by vincentvanhees** [comment]
Hi Sophie, Where can I find the better documentation you are referring to?

**Comment by sarnoult** [comment]
Hi Vincent,

you can access the requirements [here](https://github.com/cltl/vu-rm-pip3/blob/master/docs/requirements.md) (also accessible from Readme/Further reading). 
There is also a Dockerfile now, but there may be some overlap between it and python's requirements.txt.
Comments and corrections are welcome.

**Comment by vincentvanhees** [comment]
Thanks Sophie, I have now managed to get the pipeline to run. Here is my initial feedback on the documentation (without having read everything in detail yet):
1. The installation instruction is clear now. Possibly it is worth trying to make it even more accessible for a non-tech savvy end-users. For example, we could elaborate a little bit on what a Python environment is when stating "Within the python environment of your choice, do.." and in the section about component dependencies it may help to provide the inexperienced user with some guidance on how to know whether a dependency is missing and where to get them when they are missing. Possibly there are tutorials elsewhere on the web, so we could simply refer to those.
2. "The NewsReader pipeline we use here is a version of the Dutch NewsReader pipeline." links to your own [documentation file](https://github.com/cltl/vu-rm-pip3/blob/master/docs/newsreader.md). Is there no external link to the real original Newsreader pipeline developed in the Newsreader project? Maybe you could make it more explicit what parts of this pipeline are identical to the original pipeline and which parts are not?
4. The first, second and fourth sentence of the README heavily overlap. Would be good to rephrase this to make it more readable.
5. Further, it would be good to have a section in the README where you detail the developer instructions. Do you welcome contributions? If so, what are the rules?

**Comment by sarnoult** [comment]
Thanks for the comments, they are largely addressed by commit d6c84f7.
